### PR TITLE
Update tmr-meeting-recording-change.md

### PR DESCRIPTION
**Pull request by devikabhagwats**
Added the following comment for channel meeting recording storage to avoid several ICMs

Channel Meeting Recording upload to SP is not supported for IP based restrictions (we are working to enable it). Meanwhile using simple conditional access is recommended.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/tmr-meeting-recording-change.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ Set-CsTeamsMeetingPolicy -Identity Global -RecordingStorageMode "Stream"
...
 |Adhoc/Scheduled meeting with external users|Other meeting member   |Member who clicked on Record                                  |Member who clicked on Record has full rights to the recording. Organizer has edit rights and can share. <br /><br /> All other members of the meeting from the same tenant as the organizer have read access. <br /><br />All other external members have no access, and the Organizer must share it to them.|
-|Channel meeting                            |Channel Member         |Teams' SharePoint location for that channel                   |Member who clicked on Record has edit rights to the recording. <br /> <br />Every other member’s permissions are based on the Channel SharePoint permissions.|
+|Channel meeting                            |Channel Member         |Teams' SharePoint location for that channel. Channel Meeting Recording upload to SP is not supported for IP based restrictions (we are working to enable it). Meanwhile using simple conditional access is recommended.                   |Member who clicked on Record has edit rights to the recording. <br /> <br />Every other member’s permissions are based on the Channel SharePoint permissions.|
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:121]:
```suggestion
|Channel meeting                            |Channel Member         |Teams' SharePoint location for that channel. Channel Meeting Recording upload to SharePoint is not supported for IP-based restrictions (we are working to enable it). Meanwhile, we recommend using simple conditional access.                   |Member who clicked on Record has edit rights to the recording. <br /> <br />Every other member’s permissions are based on the Channel SharePoint permissions.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @devikabhagwats, Kindly commit my suggested changes. Many thanks.

File: Teams/tmr-meeting-recording-change.md (hunk 1 above)
Comment (devikabhagwats) [reply Teams/tmr-meeting-recording-change.md:121]: These recommendations are good. Thanks @JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by devikabhagwats** [comment]
Are there any actio9ns on me here?

File: Teams/tmr-meeting-recording-change.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ Set-CsTeamsMeetingPolicy -Identity Global -RecordingStorageMode "Stream"
...
 |Adhoc/Scheduled meeting with external users|Other meeting member   |Member who clicked on Record                                  |Member who clicked on Record has full rights to the recording. Organizer has edit rights and can share. <br /><br /> All other members of the meeting from the same tenant as the organizer have read access. <br /><br />All other external members have no access, and the Organizer must share it to them.|
-|Channel meeting                            |Channel Member         |Teams' SharePoint location for that channel                   |Member who clicked on Record has edit rights to the recording. <br /> <br />Every other member’s permissions are based on the Channel SharePoint permissions.|
+|Channel meeting                            |Channel Member         |Teams' SharePoint location for that channel. Channel meeting recording upload to SharePoint is not supported for IP-based restrictions (we are working to enable it). Meanwhile, we recommend using simple conditional access.                   |Member who clicked on Record has edit rights to the recording. <br /> <br />Every other member’s permissions are based on the Channel SharePoint permissions.|
```
Comment (devikabhagwats) [inline Teams/tmr-meeting-recording-change.md:121]: Can we pls make some more edits here for additional clarity- 

Channel meeting recording upload to SharePoint is not supported for IP-based restrictions. We recommend using Azure conditional access.      https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/overview

Comment (cichur) [reply Teams/tmr-meeting-recording-change.md:121]: hi @devikabhagwats Will you please save a file with your changes? I'll create a local branch under my login and make the changes there. Cindy

Comment (devikabhagwats) [reply Teams/tmr-meeting-recording-change.md:121]: Okay sounds good

Comment (devikabhagwats) [reply Teams/tmr-meeting-recording-change.md:121]: Any more actions on me?

Comment (cichur) [reply Teams/tmr-meeting-recording-change.md:121]: Hi @devikabhagwats,  Are you an MS employee? I sent a chat message to a Devika a few minutes ago.

Comment (devikabhagwats) [reply Teams/tmr-meeting-recording-change.md:121]: Yes I am MS employee. I just responded to your chat message. Thanks for proactive response. Let us get this corrected and out asap. Several customers are creating ICMs on this.

**Comment by tiburd** [comment]
This change has been added via a different PR. Closing this PR.

cc: @scanum

**Pull request closed without merging**